setup.py

```diff
@@ -34,6 +34,9 @@ def run(self):
     "urllib3>=1.24.2",
     # AWS
     "boto3>=1.9.0,<1.10",
+    # GCS
+    "google-cloud-storage<1.29",
+    "google-api-core<1.18",
```

Comment:
How did you decide to constrict the versions like this? 

I think if you don't have a good reason to (i.e. I know api breaking changes occurred at this version), then youo shouldn't constrain the versions.